releases/index.md

```diff
@@ -115,12 +124,18 @@ The following binaries are not suitable for production environments:
 </section>
 
 <section class="filter-content" data-scope="mac">
+
+    <p>ARM 64-bit binaries are <b>experimental</b> and not yet qualified for production use. They provide support for Mac computers with <a href="https://support.apple.com/en-us/HT211814">Apple Silicon</a>.</p>
```

Comment:
I took a look at how this looks in the rendered site and my thought on classifying the Apple silicon ARM64 binaries as `experimental` shifted. Seeing how it is presented, I think we _should not_ list the mac binaries as experimental like this. It doesn't make sense for these reasons:

- These binaries will never become qualified for production use (unless Apple shifts their stance on running apple hardware and software in the data center). This phrasing indicates they will at some point become approved for production use.
    - My interpretation of "production" is that it involves running a cockroach cluster that serves production data.
- These binaries are meant to be run by developers on their Apple silicon computers. Using `experimental` to describe them may cause confusion for them about whether they are ready to be used for this purpose. (It would certainly seem odd to me if I were looking for these binaries for development purposes.)
- We're likely never going to run automated tests in CI for macs. Instead, our developers are running tests and binaries locally regularly. This is the extent of the testing we have ever done for Intel macs and likely the extent we'll ever do for Apple silicon macs. 

Instead, I think we should present the ARM64 mac binaries the same way we present and have been presenting the Intel mac binaries.

Reply:
I've reworked this content, PTAL.

Reply:
This looks great! Thank you.